src/idl_gen_js_ts.cpp

```diff
@@ -2063,6 +2066,27 @@ class JsTsGenerator : public BaseGenerator {
       }
     }
 
+    if (!struct_def.fixed && parser_.services_.vec.size() != 0 &&
+        lang_.language == IDLOptions::kTs) {
+      auto name = Verbose(struct_def, "");
+      code += "/**\n";
+      code += "* GRPC METHOD ONLY, dont call\n";
+      code += "* @returns Uint8Array\n";
+      code += "*/\n";
+      code += "serialize():Uint8Array {\n";
+      code += "  return this.bb!.bytes();\n";
+      code += "}\n";
+
+      code += "/**\n";
+      code += "* GRPC METHOD ONLY, dont call\n";
+      code += "* @returns" + name + "\n";
```

Comment:
Rather than adding a comment what it returns, it's better to add the return type to function signature.